labextensions.txt

```diff
@@ -4,4 +4,3 @@
 @jupyter-widgets/jupyterlab-manager
 @pyviz/jupyterlab_pyviz
 jupyter-cytoscape
-qgrid2
```

Comment:
you'll be able to delete this file entirely

Reply:
1e6f88a